htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -1151,40 +1147,398 @@ function updateCompanyBankAccount($socid, $id, $request_data = null)
 	}
 
 	/**
-	 * @param int $id CompanyBankAccount's id
-	 * @param int $socid Thirdparty id
+	 * Delete a bank account attached to a thirdparty
+	 *
+	 * @param int $id ID of thirdparty
+	 * @param int $bankaccount_id ID of CompanyBankAccount
 	 *
 	 * @return int -1 if error 1 if correct deletion
 	 *
-	 * @url DELETE {socid}/CompanyBankAccount/{id}
+	 * @url DELETE {id}/bankaccounts/{bankaccount_id}
 	 */
-    function deleteCompanyBankAccount($id, $socid)
+	function deleteCompanyBankAccount($bankaccount_id, $id)
```

Comment:
inverted params?

Reply:
No, I have l left the same order (which is determined by the order of parameters in the @url annotation).
I have just changed the name of the parameters but yes, maybe I should switch the parameter order in the function signature. It does not change anything but maybe it is better for clarity :wink: 

Tell me what you think :slightly_smiling_face:

Reply:
ah ok, so you have to invert the @param comment to have the same order than in the function

Reply:
I have switched function signature parameters. That way, @param, @url annotations and function signature are in the same order. Make more sense :wink: